src/patternfly/demos/Masthead/templates/masthead-demo--context-selector.hbs

```diff
@@ -0,0 +1,14 @@
+{{#> context-selector context-selector--modifier="pf-m-full-height" context-selector--id=(concat masthead--id '-context-selector') context-selector--label-text="Selected project"}}
```

Comment:
did you mean to add this? Looks like this might have been one of the missing files from last milestone when we reworked the page demo template? That should be fixed and I don't see this partial used anywhere in this PR.